src/pspm_dcm.m

```diff
@@ -130,34 +154,31 @@
 %
 %__________________________________________________________________________
 % PsPM 5.1.0
-% (c) 2010-2021 PsPM Team
+% (c) 2010-2021 Dominik Bach (Wellcome Centre for Human Neuroimaging, UCL)
 
-%% Initialise & set output
-% ------------------------------------------------------------------------
+%% 1 Initialise & set output
 global settings;
-if isempty(settings), pspm_init; end;
-
+if isempty(settings), pspm_init; end
 dcm = [];
-
 % cell array which saves all the warnings which are not followed
 % by a `return` function
 warnings = {};
 
-%% Check input arguments & set defaults
-% -------------------------------------------------------------------------
+%% 2 Check input arguments & set defaults
 if nargin < 1
-  warning('ID:invalid_input', 'No data to work on.'); fn = []; return;
```

Comment:
I think the script has been terminated in this case and no outputs will be provided... thus we would not need to redefine fn again here.